From 8bcdddcaa612feb05c4ae5033bf9440b046ef63e Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 8 Aug 2024 10:56:16 +0100
Subject: [PATCH] docs: Restate position on removal of VMware Tools

With the "reboot after every step" change made in the previous commit,
it seems removal of VMware Tools is now more reliable.  Update the
notes about this.

Updates: commit c952f310c902e438a8b0b5240a4b486b698bede8
(cherry picked from commit e9c0d63b22bcb4c31cfaac00fc1271713c8bd3b3)
---
 docs/virt-v2v.pod | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/virt-v2v.pod b/docs/virt-v2v.pod
index 7912c6b1..50285cad 100644
--- a/docs/virt-v2v.pod
+++ b/docs/virt-v2v.pod
@@ -1257,9 +1257,8 @@ Virt-v2v attempts to remove VMware Tools.  For Windows guests this is
 supposed to happen during the first boot after conversion.
 
 We use VMware's recommended uninstallation method as that is the
-safest choice.  However our experience has shown that this method
-usually fails.  If so, VMware Tools must be removed by some other
-method.
+safest choice.  If this fails, VMware Tools must be manually removed
+by some other method.
 
 One possible method is described here:
 L<https://gist.github.com/broestls/f872872a00acee2fca02017160840624>
